### ISSUE TITLE
Crash when resize, move or use right mouse click on Ubuntu 20

### ISSUE THREAD

**Issue by terrar** [gnome-terminator/terminator#437]
Before opening an issue, please try starting terminator with an empty config
file, by issuing `terminator -g /dev/null` and see if it starts normally.
I did, same error

If it does, and you still want to open the bug, then please attach the
config file, usually found at `$HOME/.config/terminator/config` to the issue
`[global_config]
  hide_on_lose_focus = True 
[keybindings]
[profiles]
  [[default]]
    cursor_color = "#aaaaaa"
    scrollback_lines = 1000
[layouts]
  [[default]]
    [[[child1]]]
      parent = window0
      type = Terminal
    [[[window0]]]
      parent = ""
      type = Window
[plugins]
`

Fill out as many of these fields as you can

**Describe the bug**
Terminator crashes when resize or move window or use right mouse click on Ubuntu 20 (upgraded)
Tried 1.91 from Ubuntu repository and 2.01 from ppa

**To Reproduce**
Open terminator, 
try to move the window,
try to resize the window,
try to use right mouseklick in the window
click somewhere beside the window

**Expected behavior**
To work without crashing ;-)

**Screenshots**
no screenshots useful

**Desktop (please complete the following information):**
 - OS or Linux Distribution: [Ubuntu Desktop LTS]
 - Version [e.g. 20.04]

**Additional context**
Upgraded Ubuntu from 18.04 before, where terminator worked as expected.
Since Ubuntu 20 V1.9.1 did not work anymore
Today i tried to update to V2.0.1 with using the ppa same error.
Did not find anything to solve this myselve.

**Comment by mattrose** [comment]
I'm not seeing that at all.  

Can you try: `terminator -d -u -g /dev/null` and paste the output here?

**Comment by lexdene** [comment]
I have the same crash problem with this issue, but I can not reproduce it with `terminator -d -u -g /dev/null` command.
I guess it is related to my config, so I deleted `~/.config/terminator/config` file and try again, `terminator` no longer crashes.
The `~/.config/terminator/config` file is created by an older version of `terminator`. I'm not sure but it's very likely `terminator 1.91`.

After `bisect deletion` of the config file, I locate one line of the config file which is very likely the cause of the crash.

    [keybindings]
      go_next = None

I deleted the config file, launched `terminator` and deleted the key binding of `go_next`, the config file created by new version of `terminator` looks like:

    [keybindings]
      go_next = ""

Hope these details help anyone come to this issue in the future.
